apache/superset
# fix(Explore): Apply RLS at column values

### SUMMARY
When applying a filter, column values would show all possible values without respecting RLS filters. This PR fixes that.

### AFTER

https://github.com/user-attachments/assets/da96b25c-b3ba-46bb-beec-df09bb6ed6ee

### TESTING INSTRUCTIONS
1. Apply RLS filters to an account
2. Create a new chart using the target dataset
3. Create a filter on a Table chart
4. The user should only see the values of the column with RLS applied

### ADDITIONAL INFORMATION
